doc/website/getting-started/overview.md

```diff
@@ -11,637 +10,242 @@ runtime and create _publishers_ and _subscribers_. The publishers send data of a
 ```cpp
 iox::popo::Subscriber<CounterTopic> subscriber({"Group", "Instance", "CounterTopic"});
-subscriber.subscribe();
 ```
-The template data type and the three string identifiers have to match those of the publisher, in other words the
-service descriptions have to be the same (otherwise we will not receive data from our publisher).
-
-We immediately subscribe here, but this can be postponed to the point where we actually want to receive data.
-
-#### Receiving data
-For simplicity we assume that we periodically check for new data. It is also possible to explicitly wait for data
-using the [Waitset](../../../iceoryx_examples/waitset/README.md). The code to receive the data is the same, the only
-difference is the way we wake up before checking for data.
-
-##### 1. Take and process a sample
-
+Now we can use the subscriber to receive data. For simplicity, we assume that we periodically check for new data. It 
+is also possible to explicitly wait for data using the 
+[Waitset](https://github.com/eclipse-iceoryx/iceoryx/blob/master/iceoryx_examples/waitset/README.md) or the 
+[Listener](https://github.com/eclipse-iceoryx/iceoryx/blob/master/iceoryx_examples/callbacks/README.md). The code to 
+receive the data is the same, the only difference is the way we wake up before checking for data.
 ```cpp
 while (keepRunning)
 {
     // wait for new data (either sleep and wake up periodically or by notification from the waitset)
 
-    auto result = subscriber->take();
+    auto result = subscriber.take();
 
     if(!result.has_error())
     {
-        auto& maybeSample = result.value();
-        if (maybeSample.has_value())
-        {
-            auto& sample = maybeSample.value();
-            const CounterTopic* counter = sample.get();
-            //process the data
-
-            //alternatively use operator->
-            uint32_t counter = sample->counter;
-        }
-        else
-        {
-            // we received no data
-        }
-    } else {
-        iox::popo::ChunkReceiveResult& error = result.get_error();
+        auto& sample = result.value();
+        uint32_t counter = sample->counter;
+        //process the data
+    }
+    else
+    {
         //handle the error
     }
 }
-
 ```
+By calling ``take`` we get an ``expected`` and hence we have to handle the potential error.
 
-By calling ``take`` we get a ``iox::cxx::expected<iox::optional<iox::popo::Sample<const CounterTopic>>>``. Since this
-may fail, we have to handle a potential error. If there is no error, we may still have not recived a sample, indicated by
-a ``nullopt``. Otherwise we can get the pointer of our data from the sample and process the received data.
-
-We can shorten this somewhat by using a functional approach.
-
-##### 2. Functional approach
-
-```cpp
-while (keepRunning)
-{
-    // wait for new data (either sleep and wake up periodically or by notification from the waitset)
+And that's it. We have created our first simple iceoryx example.
```

Comment:
Add a link to the `icehello` example here? Will be available with #655

Reply:
Nope sinsce this is not the icehello example you introduced meanwhile. I'll change the name.